### PR TITLE
IA-2394 IA-2339 remove falco

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-2394

1. change leo logs for std to json which is streamed to SD for k8s deployments. This shouldn't have any impact on kibana logs becuz kibana streams from syslog
2. upgrade swagger ui. Tested in fiab and things work as expected
3. remove falco

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/SwaggerRoutes.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ import org.broadinstitute.dsde.workbench.leonardo.config.SwaggerConfig
 import akka.stream.scaladsl.Flow
 
 class SwaggerRoutes(swaggerConfig: SwaggerConfig) extends LazyLogging {
-  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/3.32.1"
+  private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/3.37.2"
```
Comment (rtitle) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/SwaggerRoutes.scala:14]: Whoops, good catch

**Review by rtitle — APPROVED** [review]
Looks fine, may want to build a custom image with this branch to make sure it works

**Comment by Qi77Qi** [comment]
looks like flaky test..jenkins retest

**Comment by Qi77Qi** [comment]
Tested with https://github.com/DataBiosphere/leonardo/pull/1770 which generated from https://fc-jenkins.dsp-techops.broadinstitute.org/view/Leonardo/job/leonardo-custom-image/793/

**Comment by Qi77Qi** [comment]
weird mysql connection not available error...jenkins retest

**Comment by Qi77Qi** [comment]
```
{"causes":[],"message":"Sam call to HttpMethod(DELETE) /api/resources/v1/billing-project/gpalloc-qa-master-lk3pcrb failed with error HttpEntity.Strict(application/json,2730 bytes total)","source":"rawls","stackTrace":[],"statusCode":500}
```

jenkins retest

**Pull request merged**